### DISCUSSION TITLE
Should we always list all 4 variants of temperature units (C,F,R,K)?

### DISCUSSION THREAD

**Discussion by steveraysteveray** [qudt/qudt-public-repo#1453] [Ideas]
For example, we are missing MOL-DEG_F. (we have MOL-DEG_C). Also missing MOL_K and MOL-DEG_R.

Current proposal:
Let's do all 4 wherever such a unit appears, including derived units

**Comment by jhodgesatmb** [comment]
When I was doing the IEC61360 work I noticed this and wondered...I agree.

Jack Hodges, Ph.D.
Arbor Studios

**Comment by ralphtq** [comment]
Agree
